backend/associations/serializers/media.py

```diff
@@ -19,3 +19,17 @@ class Meta:
             "tags",
         )
         fields = read_only_fields + ("name", "description")
+
+    def to_representation(self, instance):
+        res = super(MediaSerializer, self).to_representation(instance)
+
+        # Convert the server URI to an absolute URL
+        # ie /media/file.txt -> https://domain.name/media.file.txt
+        request = self.context["request"]
+        if request:
+            host = request.get_host()
+            res["url"] = "{}://{}{}".format(
+                "https" if request.is_secure() else "http", host, res["url"]
+            )
```

Comment:
Easier to read like this:

```suggestion
            res["url"] = f"{'https' if request.is_secure() else 'http'}://{host}{res['url']}"
```